src/worker.cc

```diff
@@ -1088,11 +1084,6 @@ void ParseApi(const http_request_t& request, valhalla::Api& api) {
     options.set_action(action);
   }
 
-  // disable analytics
```

Comment:
previously we had a request option relying on the do not track header being in the request. but now that we dont track anything anyway its superfluous